packages.json

```diff
@@ -22701,5 +22701,18 @@
     "description": "Fast and unsafe byte buffering for intensive network data transfer.",
     "license": "MIT",
     "web": "https://github.com/jakubDoka/netbuff"
+  },
+  {
+    "name": "geom",
```

Comment:
This name needs to match name of package which appears to be mlokmath not `geom`.